<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shreyUI</class>
 <widget class="QMainWindow" name="shreyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1904</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="blackbackground">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>-70</y>
      <width>2001</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/Black_Template.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif1">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>50</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/Iron_Template_1.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>981</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ExtraGui/Jarvis_Gui (2).gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif3">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>550</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>VoiceReg/__1.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>630</y>
      <width>521</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ExtraGui/Hero_Template.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif5">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>110</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ExtraGui/Earth_Template.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_time">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>380</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
font: 16pt &quot;MV Boli&quot;;
color: rgb(0, 255, 255);
border-radius:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_time_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>360</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/gggf.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_date_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>490</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/gggf.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_day_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>620</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/gggf.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_date">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>500</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
font: 15pt &quot;MV Boli&quot;;
color: rgb(0, 255, 255);
border-radius:none;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_day">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>640</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
font: 16pt &quot;MV Boli&quot;;
color: rgb(0, 255, 255);
border-radius:none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>480</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="start_button_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>460</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Buttons/Start.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>610</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exit_button_2">
    <property name="geometry">
     <rect>
      <x>1514</x>
      <y>610</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Buttons/Quit.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_temp">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>770</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
font: 16pt &quot;MV Boli&quot;;
color: rgb(0, 255, 255);
border-radius:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_temp_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>750</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>B.G/gggf.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>blackbackground</zorder>
   <zorder>gif1</zorder>
   <zorder>gif2</zorder>
   <zorder>gif3</zorder>
   <zorder>gif4</zorder>
   <zorder>gif5</zorder>
   <zorder>text_time_2</zorder>
   <zorder>text_date_3</zorder>
   <zorder>text_day_2</zorder>
   <zorder>start_button_2</zorder>
   <zorder>text_temp_2</zorder>
   <zorder>exit_button_2</zorder>
   <zorder>exit_button</zorder>
   <zorder>start_button</zorder>
   <zorder>text_time</zorder>
   <zorder>text_temp</zorder>
   <zorder>text_day</zorder>
   <zorder>text_date</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1904</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
